features/show_status_of_application.feature

```diff
@@ -22,5 +22,5 @@ Feature: As an Admin
     And I am on the list applications page
     Then I should see "2" applications
     And I should see t("membership_applications.index.status")
-    And I should see t("membership_applications.pending")
-    And I should see t("membership_applications.accepted")
+    And I should see "Inlämnad"
+    And I should see "Godkänd"
```

Comment:
Why were these translations removed and Swedish substituted? (this was in the prior PR as well ....)